### ISSUE TITLE
Pinpoint commits where error occurred

### ISSUE THREAD

**Issue by sdemos** [coreos/tailor#39]
This comes up in particularly large commits like the rust rewrites of coreos-metadata and update-ssh-keys, but I think giving more guidance as to which commit broke which rules would be pretty useful. 

seems related to #25 but I thought I would add a new issue that was more specific.

**Comment by crawford** [comment]
I agree this would be great. I'm not sure how Tailor would determine this from it's expression language though. Maybe it would be possible to hold back-references to the underlying context where a rule failed.
